cmd/rofl/deploy.go

```diff
@@ -4,9 +4,12 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/google/uuid"
```

Comment:
This seems duplicated now, remove this one.

Reply:
Removed.